diablo/jobs/kaltura_job.py

```diff
@@ -56,31 +58,36 @@ def description(cls):
 
 
 def _schedule_recordings(all_approvals, course):
+    term_id = course['termId']
+    section_id = int(course['sectionId'])
     all_approvals.sort(key=lambda a: a.created_at.isoformat())
     approval = all_approvals[-1]
+
     room = Room.get_room(approval.room_id)
+    meeting_days, meeting_start_time, meeting_end_time = SisSection.get_meeting_times(
+        term_id=term_id,
+        section_id=section_id,
+    )
+    # At UC Berkeley, recording starts 7 minutes after official start and ends 2 minutes after official end time.
```

Comment:
@johncrossman Perhaps this is a long-standing and permanent rule, but would it make sense to make these start and end time offsets configurable?

Reply:
@lyttam Good suggestion. It is a long-standing standard but this is a new age. I'll make 'em configurable with next PR.